usr/src/uts/common/vm/vm_page.c

```diff
@@ -5685,7 +5685,8 @@ page_migrate(
 	}
 }
 
-#define	MAX_CNT	60	/* max num of iterations */
+static uint_t reclaim_maxcnt = 60; /* max total iterations */
```

Comment:
It's not a restrict per se. Just that when static is there the compiler is more than allowed to inline all references to it and remove the symbol which is something gcc does, thus defeating attempts to be able to tune it.